web-services/query/src/main/java/datawave/webservice/query/runner/QueryExecutorBean.java

```diff
@@ -1956,9 +1960,12 @@ private VoidResponse close(String id, Principal principal) {
                     RunningQuery query = getQueryById(id, principal);
                     close(query);
```

Comment:
maybe it would be worth checking the cache before doing any additional work, but it seems either way will work.

Reply:
I debated that, however there are several steps that occur and I wanted at least an administrator to attempt the close process all the way through in case something did not take.  Also there are some subtle differences between the cancel, close, and remove that in fact I am a little confused about.  This seemed the safest solution.